app/src/main/java/org/gdg/frisbee/android/activity/PulseActivity.java

```diff
@@ -87,30 +84,30 @@ public void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setHomeButtonEnabled(true);
 
         mFetchGlobalPulseTask = mClient.getPulse(new Response.Listener<Pulse>() {
-                @Override
-                public void onResponse(final Pulse pulse) {
-                    getSupportActionBar().setListNavigationCallbacks(mSpinnerAdapter, PulseActivity.this);
-                    App.getInstance().getModelCache().putAsync("pulse_global", pulse, DateTime.now().plusDays(1), new ModelCache.CachePutListener() {
-                         @Override
-                         public void onPutIntoCache() {
-                            mPulseTargets.addAll(pulse.keySet());
-                            initSpinner();
-                         }
-                    });
-                }
-            }, new Response.ErrorListener() {
-                @Override
-                public void onErrorResponse(VolleyError volleyError) {
-                    Crouton.makeText(PulseActivity.this, getString(R.string.fetch_chapters_failed), Style.ALERT).show();
-                    Timber.e("Could'nt fetch chapter list", volleyError);
-                }
-            }
+                                                     @Override
+                                                     public void onResponse(final Pulse pulse) {
+                                                         getSupportActionBar().setListNavigationCallbacks(mSpinnerAdapter, PulseActivity.this);
+                                                         App.getInstance().getModelCache().putAsync("pulse_global", pulse, DateTime.now().plusDays(1), new ModelCache.CachePutListener() {
+                                                             @Override
+                                                             public void onPutIntoCache() {
+                                                                 mPulseTargets.addAll(pulse.keySet());
+                                                                 initSpinner();
+                                                             }
+                                                         });
+                                                     }
+                                                 }, new Response.ErrorListener() {
+                                                     @Override
+                                                     public void onErrorResponse(VolleyError volleyError) {
+                                                         Crouton.makeText(PulseActivity.this, getString(R.string.fetch_chapters_failed), Style.ALERT).show();
+                                                         Timber.e("Could'nt fetch chapter list", volleyError);
+                                                     }
+                                                 }

```

Comment:
I'm using Novoda's settings, which apparently are not correct. Do you know which setting I need to change? Maybe it's better that we ignore this for now and once I have the correct settings apply reformatting to all the class files in one single commit.